### PR TITLE
Tweaks to PSE

### PR DESCRIPTION
**Pull request by pawelpabich**
This is PR tweaks 3 things:
* It replaces future `maybe` solutions with examples from the past that should be much easier to understand
* It shfits focus from working with 1 team to working with many teams which is what's required to introduce any significant change at Octopus
* It emphasises the significance of the initiatives we expect from L5s. We don't expect them to patch things here and there, we expect them to take us to the next level.

### CODE REVIEW

File: Software-Engineering.md [hunk 1]
```diff
@@ -142,13 +142,13 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
...
 - I collaborate with others to articulate solutions, positively influence and convince that we should fund their implementation
-- My contributions leave our Engineering team, and Octopus as a whole, in a stronger strategic position
+- My contributions are strategic and take our Engineering team, and Octopus as a whole, to the next level
 
 ### I am trusted to take care of a large, critical or complex strategic area or product
 
-- I am comfortable taking on a major challenge (e.g. re-write Octopus Server, or re-architect Octopus Cloud using microservices)
-- I identify team members with the necessary skills, work out how to fill gaps, and guide and influence that team (or teams) collaboratively
-- I use my Engineering experience to collaboratively steer teams toward good decisions along the way
+- I am comfortable taking on a major challenge that requires to build or re-architect an important part of Octopus ecosystem (e.g. Modularity, Octopus Cloud)
```
Comment (Waldo000000) [inline Software-Engineering.md:149]: Suggest phrasing such that new starters can understand, e.g.:

```suggestion
- I am comfortable taking on a major challenge that requires building or re-architecting an important part of the Octopus ecosystem (e.g. re-architecting Octopus Server to allow for plugin functionality, or creating a new Octopus Cloud platform)
```

File: Software-Engineering.md [hunk 2]
```diff
@@ -142,13 +142,13 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
...
+- I identify teams with the necessary skills, work out how to fill gaps, and guide and influence these teams collaboratively
```
Comment (Waldo000000) [inline Software-Engineering.md:150]: I imagine finding ways to persuade for the necessary teams to be created (e.g. via internally recruitment or restructuring) is important, too?

```suggestion
- I identify people and teams with the necessary skills, work out how to fill gaps, and guide and influence these people and teams collaboratively
```

File: Software-Engineering.md [hunk 3]
```diff
@@ -142,13 +142,13 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I have a deep understanding of Octopus products and our strategy
 - I identify significant problems, inhibitors and opportunities, and advocate for solutions that align our strategy and Engineering efforts
 - I collaborate with others to articulate solutions, positively influence and convince that we should fund their implementation
-- My contributions leave our Engineering team, and Octopus as a whole, in a stronger strategic position
+- My contributions are strategic and take our Engineering team, and Octopus as a whole, to the next level
```
Comment (Waldo000000) [inline Software-Engineering.md:145]: Can we clarify what "next level" means?

**Review by Waldo000000 — APPROVED** [review]
👍 With comments

File: Software-Engineering.md (hunk 3 above)
Comment (pawelpabich) [reply Software-Engineering.md:145]: I agree this is vague. 
That being said, this seems to be one of the terms that people don't have a problem to understand. I've used it multiple times and it didn't cause any confusion. In other words, I think people should not have a problem to assess whether an idea is taking Octopus to the next level or not.

I would like to make it more concrete but I'm not sure how.  
Do you have any suggestions?

Comment (Waldo000000) [reply Software-Engineering.md:145]: > Do you have any suggestions?

Yes. When I think "take to the next level", I interpret that as meaning "leave in a stronger strategic position" 😄

File: Software-Engineering.md (hunk 1 above)
Comment (pawelpabich) [reply Software-Engineering.md:149]: > Suggest phrasing such that new starters can understand, e.g.:

My goal is to be as specific here as possible.  Every company has a plugin system which does something else but there has been only one Modularity project at Octopus.  I would expect new people to learn about the project and understand why it was important for us.

File: Software-Engineering.md (hunk 3 above)
Comment (pawelpabich) [reply Software-Engineering.md:145]: This is a PR for a branch so we can evolve it in the PSE branch.

Comment (pawelpabich) [reply Software-Engineering.md:145]: > Yes. When I think "take to the next level", I interpret that as meaning "leave in a stronger strategic position

The same level of vagueness :)

**Pull request merged**